LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Database/DataBrokerProtectionDataManager.swift

```diff
@@ -132,11 +133,16 @@ extension DataBrokerProtectionDataManager: InMemoryDataCacheDelegate {
 
         delegate?.dataBrokerProtectionDataManagerDidDeleteData()
     }
+
+    public func willOpenSendFeedbackForm() {
+        delegate?.dataBrokerProtectionDataManagerWillOpenSendFeedbackForm()
+    }
 }
 
 public protocol InMemoryDataCacheDelegate: AnyObject {
     func saveCachedProfileToDatabase(_ profile: DataBrokerProtectionProfile) async throws
     func removeAllData() throws
+    func willOpenSendFeedbackForm()
```

Comment:
This being part of the `InMemoryDataCacheDelegate` feels weird. It should belong to another delegate.

I know this is because of the DBP architecture, and does not need to change now, but it is something that I wanted to call out.

Reply:
Agreed. This feels bad to write, thanks for call out.